src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Restore-AzsBackup.ps1

```diff
@@ -58,6 +61,11 @@ function Restore-AzsBackup {
         [System.String]
         $Location,
 
+        [Parameter(Mandatory = $false, ParameterSetName = 'Backups_Restore')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'ResourceId')]
+        [System.String]
+        $RoleName = "-",
```

Comment:
This default value might be confusing to customers. How about we don't set $RestoreOptions.RoleName when $RoleName is null or empty?

Reply:
sure